picasso/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
 dependencies {
   api deps.okhttp
+  implementation deps.androidSvg
```

Comment:
Should this be included in a separate optional dependency so we don't impose this library on all users?